### PR TITLE
Flake8 on CI

### LINKED ISSUE #60: Enable flake8 check
Enable flake8 checks at https://github.com/vmck/vmck/blob/209b00c80a7f997bcc780530a9f947db92337572/.drone.yml#L9 and fix any errors that may arise. Use a branch on this repository to have the CI at https://jenkins.liquiddemo.org/vmck/vmck/ do what you want.

### PR DESCRIPTION
**Pull request by Jokeswar**
Django might brake flake8 checks if it migrates a new model or updates an old one

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ steps:
   image: python:3.7-stretch
   commands:
   - pip install flake8 > /dev/null
-  - flake8 || true  # not now
+  - flake8   # not now
```
Comment (gabriel-v) [inline .drone.yml]: Kill the comment too since `now` is exactly the time

File: contrib/build.py [hunk 2]
```diff
@@ -21,7 +21,12 @@
     'bionic-server-cloudimg-amd64.img'
 )
 
-SSH_PUBKEY = 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ== vagrant insecure public key'
+SSH_PUBKEY = 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eW \
```
Comment (gabriel-v) [inline contrib/build.py]: Escaping this like that looks like you might insert some spaces. Leave this unchanged, append `# noqa`

File: testsuite/settings.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 import os
 from datetime import datetime
 from vmck.utils import is_true, random_code
-from vmck.base_settings import *
```
Comment (gabriel-v) [inline testsuite/settings.py]: Leave this here, it was pulling the default settings.

You can ` # noqa: E123,E456` after this line.

File: vmck/migrations/0001_initial.py [hunk 4]
```diff
@@ -15,21 +15,27 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Artifact',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('id', models.AutoField(auto_created=True, primary_key=True,
```
Comment (gabriel-v) [inline vmck/migrations/0001_initial.py]: Please set up [one of these](https://github.com/hoover/snoop2/blob/d539509681b2ad01a100ed4d2c64cd0c7ece513b/.flake8#L7) and ignore the line length for migrations, so future migrations won't crash flake8.

File: vmck/migrations/0002_source_upload.py [hunk 5]
```diff
@@ -14,16 +14,20 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Source',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('id', models.AutoField(auto_created=True, primary_key=True,
```
Comment (gabriel-v) [inline vmck/migrations/0002_source_upload.py]: Also, please revert all all line length-related changes to migration files.

File: vmck/settings.py [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import os
 import sentry_sdk
 from .utils import is_true
-from .base_settings import *
```
Comment (gabriel-v) [inline vmck/settings.py]: Leave (and ignore) this star import, we wouldn't want to update this list too on every change in that file.

File: contrib/build.py [hunk 7]
```diff
@@ -21,7 +21,7 @@
     'bionic-server-cloudimg-amd64.img'
 )
 
-SSH_PUBKEY = 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ== vagrant insecure public key'
+SSH_PUBKEY = 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ == vagrant insecure public key'  # noqa: E501
```
Comment (mgax) [inline contrib/build.py]: Are you sure it's ok to add a space before `==`? This is base64 after all.

File: testsuite/settings.py (hunk 3 above)
Comment (mgax) [reply testsuite/settings.py]: You can just say `noqa`, we understand that importing settings from a defaults file is a slightly-non-pythonic operation that is ok for this use case.

Comment (mgax) [reply testsuite/settings.py]: Umm, you marked as resolved, but did not change the line :) At least write a message saying "i refuse to make this change" :p

Comment (Jokeswar) [reply testsuite/settings.py]: Umm, I readded the line with # noqa in my seccond commit. Maybe I don't understand what you want me to change?

**Pull request merged**